core-web/libs/edit-content/src/lib/fields/dot-edit-content-relationship-field/components/dot-select-existing-content/components/search/search.component.ts

```diff
@@ -46,6 +49,16 @@ export class SearchComponent {
      */
     $overlayPanel = viewChild(OverlayPanel);
 
+    /**
+     * Reference to the language field component to access its store.
+     */
+    $languageField = viewChild(LanguageFieldComponent);
```

Comment:
I suggest using `viewChild.required` because this component is always present in the template.